ORK1Kit/ORK1Kit/Common/ORK1WebViewStepViewController.h

```diff
@@ -50,7 +50,8 @@ ORK1_CLASS_AVAILABLE
  a web view step in a task, and present a task view controller for that task.
  */
 @interface ORK1WebViewStepViewController : ORK1StepViewController<WKScriptMessageHandler, WKNavigationDelegate>
-
+@property (nonatomic, strong, readonly) WKWebView *webView;
```

Comment:
Minor - if we're making this a @property, should we remove the `*-webView` declaration in the .m file on line 90?

Reply:
Updated.